features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/recorder/resources/DefaultImageWireframeHelper.kt

```diff
@@ -252,10 +257,17 @@ internal class DefaultImageWireframeHelper(
                     position = compoundDrawablePosition
                 )
 
+                wireframeIndex++
+                val resourceCacheKey = if (resourceIdCacheKey != null) {
+                    "$resourceIdCacheKey" + "_$wireframeIndex"
+                } else {
+                    null
+                }
```

Comment:
what's the reason this modification is only applied for CompoundDrawable?

Reply:
CompoundDrawables can have multiple drawables associated, so in the case that an override key is provided the drawables need to be differentiated in the cache but if no key is provided then it will default to the original behavior